### DISCUSSION TITLE
Parameters for the embedding net

### DISCUSSION THREAD

**Discussion by wei0py** [deepmodeling/deepmd-kit#1777] [Q&A]
Hi,

Is there any way to get the trained embedding network parameters (e.g. weights, bias) from the model.ckpt. I don't see them from the tensors output by model.ckpt like the fitting net.

Thanks,
Buyu

**Comment by njzjz** [comment]
https://docs.deepmodeling.com/projects/deepmd/en/master/api_py/deepmd.utils.html#deepmd.utils.graph.get_embedding_net_nodes
